src/templates/next/components/InfoCards/InfoCards.tsx

```diff
@@ -0,0 +1,116 @@
+import { InfoCardsProps } from "~/common"
+import Card from "../shared/Card"
+
+const TitleSection = ({
+  title,
+  subtitle,
+  className,
+}: {
+  title: InfoCardsProps["title"]
+  subtitle: InfoCardsProps["subtitle"]
+  className?: string
+}) => {
+  return (
+    <div className={`flex flex-col gap-8 self-start max-w-3xl ${className}`}>
+      <h3 className="text-content text-2xl leading-tight font-semibold sm:text-4xl ">
+        {title}
+      </h3>
+      {subtitle && (
+        <p className="text-content text-sm sm:text-lg">{subtitle}</p>
+      )}
+    </div>
+  )
+}
+
+const InfoCards = ({
+  sectionIdx,
+  cards,
+  title,
+  subtitle,
+  variant,
+}: InfoCardsProps) => {
+  const bgColor = sectionIdx % 2 === 1 ? "bg-gray-100" : "bg-white"
```

Comment:
not sure if we still want this alternating bg-gray behaviour for Next, I think we removed it for other homepage components like infobar and infocols cc @dcshzj

Reply:
Ah okay, have removed in https://github.com/isomerpages/isomer-components/pull/38/commits/b9090a42c9eb867b02b08434c37283ed65251b38